test/test-config.sh

```diff
@@ -163,8 +163,8 @@ fi
 # < 1.19) and because there were random connection failures to node ports during sanity
 # testing (Kubernetes 1.19.0)
 : ${TEST_FEATURE_GATES:=CSINodeInfo=true,CSIDriverRegistry=true,CSIBlockVolume=true,CSIInlineVolume=true\
-$(case ${TEST_KUBERNETES_VERSION} in 1.1[0-5]) ;; *) echo ',EndpointSlice=false';; esac)\
-$(case ${TEST_KUBERNETES_VERSION} in 1.19) echo ',CSIStorageCapacity=true,GenericEphemeralVolume=true,EndpointSliceProxying=false';; esac)\
+$(case ${TEST_KUBERNETES_VERSION} in 1.1[0-5]) ;; 1.20) ;; *) echo ',EndpointSlice=false,EndpointSliceProxying=false';; esac)\
+$(case ${TEST_KUBERNETES_VERSION} in 1.19 | 1.20) echo ',CSIStorageCapacity=true,GenericEphemeralVolume=true,EndpointSliceProxying=false';; esac)\
```

Comment:
This looks odd and/or unnecessarily complex.

The first line is responsible for adding `EndpointSlice=false` on those releases which support it - basically anything >= 1.16. The second line then added feature gates for Kubernetes = 1.19.

Why does `EndpointSliceProxying=false` get added in the first line?

How does 1.20 fit into this? Neither the commend above nor the commit message have that.

Reply:
oops, I broke it in rebasing, `EndpointSliceProxying=false` should be removed from the second line. My intention was separate endpoint features and storage features and disabling limit endpoint features to 1.19. I will fix this.